models/DataObject/Fieldcollection/Data/AbstractData.php

```diff
@@ -103,7 +103,7 @@ public function getType()
     }
 
     /**
-     * @return mixed
+     * @return Model\DataObject\Fieldcollection\Definition
```

Comment:
Definition|null

Reply:
Do you think we cannot be sure that the definition exists? Than we have to check it on any usage with an if. Now there is no if at any.

Reply:
good point!...I was just looking at return values from `getByKey` method as it still returns null. please revert the null change and sorry for rework.

Reply:
Okay, I reverted it.